sdk/search/Azure.Search.Documents/src/Models/AzureSearchDocumentsContext.Custom.cs

```diff
@@ -13,10 +13,5 @@ namespace Azure.Search.Documents
     [EditorBrowsable(EditorBrowsableState.Never)]
     public partial class AzureSearchDocumentsContext : ModelReaderWriterContext
     {
-        private AzureSearchDocumentsContext _azureSearchDocumentsContext;
```

Comment:
You can delete this file completely since its not generated in the generated folder.